modules/gdscript/gdscript_parser.cpp

```diff
@@ -2042,6 +2042,14 @@ GDScriptParser::ExpressionNode *GDScriptParser::parse_identifier(ExpressionNode
 			case SuiteNode::Local::UNDEFINED:
 				ERR_FAIL_V_MSG(nullptr, "Undefined local found.");
 		}
+	} else if (current_class != nullptr && current_class->has_member(identifier->name)) {
+		const ClassNode::Member &declaration = current_class->get_member(identifier->name);
+
+		if (declaration.type == ClassNode::Member::VARIABLE) {
+			identifier->source = IdentifierNode::MEMBER_VARIABLE;
+			identifier->variable_source = declaration.variable;
+			declaration.variable->usages++;
+		}
```

Comment:
This won't really work in all cases because member variables can be used before they are declared. This check needs to be done in the analyzer.

Reply:
You're right, thanks! Did not think about that. Moved the usage increment into the analyzer. Here's a small test case:
```gdscript
var pub

var _priv_forward_used
var _priv_forward_unused

func test1() -> void:
	pub = _priv_forward_used
	pub = _priv_backward_used
	
var _priv_backward_used
var _priv_backward_unused
```
Result:
![image](https://user-images.githubusercontent.com/4233942/131865563-83a7cd47-5230-4eef-910a-092c75cebe29.png)

I don't like, that some usages are counted in the parser and this new one in the analyzer...

Reply:
> I don't like, that usages are counted in the parser and this new one in the analyzer, but I don't think that this could be changed easily..

TBH I don't like it either, but for now I guess it's not worth changing.